File: captainhcg/js/412-fizz-buzz.js
var fizzBuzz = function(n) {
    var ret = []
    for(var i=1; i <= n; i++){
        if(i % 15 == 0){
            ret.push("FizzBuzz");
        } else if(i%3 == 0){
            ret.push("Fizz");
        } else if(i%5 == 0){
            ret.push("Buzz");
        } else {
            ret.push(i.toString());
        }
    }
    return ret
};

File: bwu/math/412-fizz-buzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        ret = []
        for i in xrange(1, n + 1):
            if i % 3 == 0 and i % 5 == 0:
                ret.append('FizzBuzz')
            elif i % 3 == 0:
                ret.append('Fizz')
            elif i % 5 == 0:
                ret.append('Buzz')
            else:
                ret.append(str(i))
                
        return ret
